check_process

```diff
@@ -20,15 +16,19 @@
 		setup_private=1
 		setup_public=1
 		upgrade=1
-		upgrade=1		from_commit=cef22ac5634565cecd2d8545229efe3fe79d85bc
+		#4.4
+		upgrade=1		from_commit=66eb909e771d11b092dfff2acf50ff63fa6f6ec6
+		#4.6.1
+		upgrade=1		from_commit=3f8363cdc588e609a775cb8b78b9d080f96405d9
 		backup_restore=1
 		multi_instance=1
-		port_already_use=0
 		change_url=1
 ;;; Options
 Email=
 Notification=all
 ;;; Upgrade options
-	; commit=cef22ac5634565cecd2d8545229efe3fe79d85bc
-		name=Upgrade from 3.11.3
+	; commit=66eb909e771d11b092dfff2acf50ff63fa6f6ec6
+		name=Upgrade from 4.4
+	; commit=3f8363cdc588e609a775cb8b78b9d080f96405d9
+		name=Upgrade from 4.6.1
```

Comment:
My policy is to add an upgrade test at each "breaking change" of the package or the app. I let the 3.11.3 here because there is a migration and a new config file to set to make the upgrade works, and I consider that if the upgrade from a very old version works, then the immediate previous upgrade should too

Reply:
Annnnnnd I'm sorry about all this story since the version 3.11.3 can't be installed, I'll restore the test upgrade from the 4.4.